### ISSUE TITLE
Openshot crashes almoste immediately after openening, particularly when dragging .mov files to the openshot browser. 

### ISSUE THREAD

**Issue by Danrancan** [OpenShot/openshot-qt#4476]
**Describe the bug**
Openshot crashes almoste immediately after openening, particularly when dragging .mov files to the openshot browser. This also happens when rearranging video files or using heavy processing power (seemingly).

**Steps to reproduce the behavior:**
1. Open up openshot
2. Drag a handful of MOV files to the openshot browser for import
3. Openshot immediately crashes and dissapears from the screen. You must reopen it to get started again, only to repeat the same errors.

**Expected behavior**
I expect openshot to not crash after dragging video files into it or rearranging content.

**System Details**
- OpenShot Version: 2.6.1-dev
- libopenshot Version:  0.2.7-dev
- Platform: Darwin-20.6.0-x86_64-i386-64bit
- Processor: i386
- Machine: x86_64
- Python version: 2.7.16
- Python3 version: 3.9.7
- Qt5 version: Unknown (how to find version of qt?)
- PyQt5 version: Unknown/not-installed?
- Qt Detected Languages: ['en-US']
- LANG Environment Variable: 
- LOCALE Environment Variable: 
- Daily Build: Verified issue still exists in daily build: http://github.com/OpenShot/openshot-qt/releases/download/OpenShot-v2.6.1-dev-daily-8361-21597c8a-73c519f0-x86_64.dmg

**Log Files**
- [openshot-qt.log](https://s3.amazonaws.com/logs.openshot.org/qtfqz22pgn0z4odu18pdjk3s2i815vc6/openshot-qt.zip) (11 KB)
- libopenshot.log: File too small or empty (not uploaded)

**Exception / Stacktrace**
```No stacktrace found in log files```

**Screenshots** (Optional)
If applicable, add screenshots to help explain your problem. You can include screenshots by
copy/pasting them on GitHub or dragging-and-dropping into the GitHub page. All images are public,
so please don't post screenshots containing personal information.

**Comment by JacksonRG** [comment]
Hi, did you have verbose logging on when created this log file? If not, you can find instructions on enabling it [here](https://github.com/OpenShot/openshot-qt/wiki/Issue-Logging).

**Comment by JacksonRG** [comment]
Additionally, from the logs it looks like the first few files import without crashing. Does importing just the first one cause a crash?

---
Possibly related issues: 
#4480
#4483

**Comment by Danrancan** [comment]
> Additionally, from the logs it looks like the first few files import without crashing. Does importing just the first one cause a crash?
> 
> Possibly related issues: #4480 #4483

I dont believe it does. But I will have to double check when I get back on my mac.

**Comment by Danrancan** [comment]
> Hi, did you have verbose logging on when created this log file? If not, you can find instructions on enabling it [here](https://github.com/OpenShot/openshot-qt/wiki/Issue-Logging).

I can't remember if I did or didn't. When I get some free time i will enable it and upload the log again.

**Comment by ddm91401** [comment]
OpenShot was also crashing on me when trying to import .MOV files that were created on my iPhone 6s.
I fixed it by using VLC to convert the .MOV files to .MP4.  In VLC, Open Media > Convert/save.  You must select the "Video for YouTube" (SD or HD) profile; the other MP4 profiles did not work for me.

**Comment by philsward** [comment]
Like @ddm91401 mentioned, I tried the VLC route to convert .MOV which did finally work, but the only mp4 file output that I was successful on was the profile:

Video for MPEG4 1080p TV/device.

I tried:
Video - H.264 + MP3 (MP4)
Video - H2.64 + MP3 (TS)

I also tried Video - H.265 + MP3 (MP4) which didn't crash, but Openshot wouldn't render the video so it was a no-go as well. Maybe I need codecs installed for H.265? Figured Openshot would have its own. Total Openshot noob here so keep that in mind.

There were other VLC convert profiles I didn't try that might work. The ones listed are the ones I did try.
